.github/workflows/wheel-builder.yml

```diff
@@ -57,10 +57,6 @@ jobs:
     strategy:
       matrix:
         PYTHON:
-          - VERSION: '2.7'
-            ABI_VERSION: '2.7'
-            DOWNLOAD_URL: 'https://www.python.org/ftp/python/2.7.17/python-2.7.17-macosx10.9.pkg'
-            BIN_PATH: '/Library/Frameworks/Python.framework/Versions/2.7/bin/python'
```

Comment:
The regex match in the build the wheel bash for macOS can be removed now and we can define `ABI_VERSION` as just `cp36`.